### ISSUE TITLE
path planner for this package

### ISSUE THREAD

**Issue by Rohitra078** [leggedrobotics/elevation_mapping_cupy#46]
I am trying to create a 3d path planner using elevation mapping and was wondering if there are any packages you use or suggest for that purpose.

**Comment by mktk1117** [comment]
Do you plan to do full 3d planning with drones? or do with ground robot?

**Comment by Rohitra078** [comment]
yeah i have a robot that can fly and also do mobile navigation and i am planning to add a factor which focus on more mobile navigation and use aerial capability when surface is non traversable
